modules/model/shared/src/main/scala/observe/model/ObserveStep.scala

```diff
@@ -19,8 +19,8 @@ import monocle.syntax.all.*
 import observe.model.dhs.*
 import observe.model.enums.*
 
-sealed trait Step extends Product with Serializable {
-  def id: StepId
+sealed trait ObserveStep extends Product with Serializable {
```

Comment:
Is it possible to just use core's `Step` instead of `ObserveStep`? Or at least have `ObserveStep` wrap a `Step` and get `id`, `instConfig` and `stepConfig` from the wrapped `Step`?

Reply:
They have very different structures as one describes the step while `ObserveStep` contains state of the current execution

Reply:
I see both share the `DynamicConfig` and `StepConfig` (and `id`).

Would it be possible for `ObserveStep` to wrap a (core) `Step`?

I'm in the process of making the server share the current atom configuration with the UI but I find that the server has `ObserveStep`s, not (core) `Step`s. The UI has all the logic in place (which is shared with explore) to display (core) `Step`s. If the server provided (core) `Step`s instead of `ObserveStep`s, it would be much easier in the UI.

Reply:
Upon closer examination, it might not be that hard to create an adaptation layer for `ObserveStep` for the UI. I'll attempt this.